corehq/apps/users/static/users/js/roles.js

```diff
@@ -335,12 +335,33 @@ hqDefine('users/js/roles',[
                     data.name = data.name.trim();
                 }
 
-                data.permissions.view_report_list = ko.utils.arrayMap(ko.utils.arrayFilter(data.reportPermissions.specific, function (report) {
-                    return report.value;
-                }), function (report) {
-                    return report.path;
-                });
-                data.permissions.view_reports = data.reportPermissions.all;
+                const unWrapParameterizedPermission = function (permission, item_attr='slug') {
```

Comment:
nitpick: `unWrap` => `unwrap`

Reply:
I thought that looked funny but couldn't put my finger on it. Thanks: a1f5de2014b